src/certs/ca/cert/v1.rs

```diff
@@ -9,10 +9,10 @@ const NAPLES_ARK_SIG: &[u8] = include_bytes!("../../../../tests/naples/ark.cert.
 const NAPLES_ARK: Preamble = Preamble {
     ver: 1u32.to_le(),
     data: Data {
-        kid: 122178821951678173525318614033703090459u128.to_le(),
-        sid: 122178821951678173525318614033703090459u128.to_le(),
+        kid: 122178821951678173525318614033703090459u128.to_le_bytes(),
+        sid: 122178821951678173525318614033703090459u128.to_le_bytes(),
```

Comment:
Do the conversion manually (try Python!). You can store the result as a byte literal `b"..."`. This removes the need for endianness conversion altogether. Which means we don't have to bump the minimum rust version either.

Reply:
I was able to get my array of bytes in little-endian, but storing the result as a byte string resulted in an array that was too large since each byte takes up 2 characters so I wound up with a [u8; 32] instead of a [u8; 16].

My latest revision just has a generated little-endian [u8; 16].

Reply:
@connorkuehl You just need to escape them. `b"\x00\x01"`

Reply:
TIL, thanks :grinning: